Pod/Classes/OAStackViewDistributionStrategy.m

```diff
@@ -90,21 +90,25 @@ - (void)alignView:(UIView*)view afterView:(UIView*)previousView {
 - (void)alignLastView:(UIView*)view {
   NSString *constraintString = [NSString stringWithFormat:@"%@:[view]-(lastMargin)-|", [self currentAxisString]];
   NSNumber *lastMargin = @([self lastMargin]);
-  [self.stackView addConstraints:
-   [NSLayoutConstraint constraintsWithVisualFormat:constraintString
-                                           options:0
-                                           metrics:NSDictionaryOfVariableBindings(lastMargin)
-                                             views:NSDictionaryOfVariableBindings(view)]];
+  id arr = [NSLayoutConstraint constraintsWithVisualFormat:constraintString
```

Comment:
Absolutely, but similar existing methods used `id arr = …` for some reason so I stuck to it. I'd be happy to change all of them